Library/Homebrew/test/language/node_spec.rb

```diff
@@ -2,28 +2,14 @@
 
 describe Language::Node do
   describe "#setup_npm_environment" do
-    it "does nothing when npmrc exists" do
-      expect(subject.setup_npm_environment).to be_nil
-    end
-
-    it "calls prepend_path when node formula exists and npmrc does not exist" do
-      node = formula "node" do
-        url "node-test"
-      end
-      stub_formula_loader(node)
-      allow_any_instance_of(Pathname).to receive(:exist?).and_return(false)
-      expect(ENV).to receive(:prepend_path)
-      subject.setup_npm_environment
```

Comment:
Looks like `ENV.prepend_path` isn't called in test either, even if I remove the now unneeded line 14.